### DISCUSSION TITLE
Query engine and migration engine binaries available on their own?

### DISCUSSION THREAD

**Discussion by andreimc** [prisma/prisma#17318] [Q&A]
previously I could download: 

```
PRISMA_VERSION="efdf9b1183dddfd4258cd181a72125755215ab7b"
OS="linux-musl"
QUERY_ENGINE_URL="https://binaries.prisma.sh/all_commits/${PRISMA_VERSION}/${OS}/query-engine.gz"
```

But after that sha it no longer works. I can build it myself etc but I am just wondering if it's provided somewhere?

**Comment by nurul3101** [comment]
Hey @andreimc 👋 

They should be available, can you check if it is maybe a network issue?

I just tried for the latest prisma version through this URL: https://binaries.prisma.sh/all_commits/d6e67a83f971b175a593ccc12e15c4a757f93ffe/linux-musl/query-engine.gz

And the binaries were downloaded as expected.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
